subprojects/execution/src/test/groovy/org/gradle/internal/execution/steps/FingerprinterFixture.groovy

```diff
@@ -40,6 +41,15 @@ abstract trait FingerprinterFixture {
         )
     )
 
+    final OutputFileCollectionFingerprinter outputFingerprinter = new OutputFileCollectionFingerprinter(
+        new DefaultFileSystemSnapshotter(
+            new TestFileHasher(),
+            new StringInterner(),
+            TestFiles.fileSystem(),
+            new DefaultFileSystemMirror(new NoWellKnownFileLocations())
```

Comment:
These objects should be reused between the two fingerprinters.

Reply:
Re-using the objects now.